agent/installer/installer.go

```diff
@@ -37,21 +37,28 @@ func getSupportedRegistry(bd *bundleDownloader, ob algo.OutputBuilder) registry
 		os   string
 		k8s  string
 		algo algo.K8sStepProvider
+		osFilter string
 	}{
-		{"Ubuntu_20.04.1_x86-64", "v1.22.1", &algo.Ubuntu20_4K8s1_22{}},
+		// Ubuntu support
+		{os: "Ubuntu_20.04.1_x86-64", k8s: "v1.22.1", algo: &algo.Ubuntu20_4K8s1_22{}},
```

Comment:
Suggestion: Can we have below instead and remove the osFilter and aliasOSMap implementation.
```suggestion
		{os: "Ubuntu_20.04.*_x86-64", k8s: "v1.22.1", algo: &algo.Ubuntu20_4K8s1_22{}},
```

Reply:
See below about aliasOSMap. Can be done post FC.

Removing patch version from bundle name will make it *impossible* to do something different for some patch version, It makes too vague what we map to what. What is the benefit of doing so?

Reply:
@ratanasovvmw This comment takes into consideration my other comment where I thought the implementation can be made simpler by defining the regex for the `os` field itself. But I am ok if we wanted to do this post 0.1.0 release.